packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -363,67 +434,271 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
+                justifyContent: 'stretch',
+                alignItems: 'stretch',
+              }}
+            >
+              <SynapsePriceTableCell sx={{ backgroundColor: '#b6cedf' }}>
+                <Typography
+                  className="no-margin"
+                  variant="body1"
+                  sx={{ fontWeight: 700 }}
+                >
+                  Standard Plan
+                </Typography>
+              </SynapsePriceTableCell>
+              <SynapsePriceTableCell sx={{ backgroundColor: '#c4d9e4' }}>
+                <Typography variant="body2" sx={{ fontWeight: 700 }}>
+                  Free
+                </Typography>
+              </SynapsePriceTableCell>
+            </Box>
+            <Box
+              sx={{
+                display: 'grid',
+                gridTemplateRows: '65px auto',
+                justifyContent: 'stretch',
+                alignItems: 'stretch',
+              }}
+            >
+              <SynapsePriceTableCell sx={{ backgroundColor: '#f0a675' }}>
+                <Typography
+                  className="no-margin"
+                  variant="body1"
+                  sx={{ fontWeight: 700 }}
+                >
+                  Managed Plan
+                </Typography>
+              </SynapsePriceTableCell>
+              <SynapsePriceTableCell sx={{ backgroundColor: '#f9d6c0' }}>
+                <Typography
+                  variant="body2"
+                  sx={{
+                    paddingTop: '20px',
+                    marginLeft: '10px',
+                    marginRight: '10px',
+                  }}
+                >
+                  <strong>
+                    Starting at $6,000/year, <br />
+                    min 5 years*{' '}
+                  </strong>
+                  <em>
+                    <br />
+                    Can be customized <br />
+                    Recommended for NIH DMSP
+                  </em>
+                </Typography>
+              </SynapsePriceTableCell>
+            </Box>
+            <Box
+              sx={{
+                display: 'grid',
+                gridTemplateRows: '65px auto',
+                justifyContent: 'stretch',
+                alignItems: 'stretch',
+              }}
+            >
+              <SynapsePriceTableCell sx={{ backgroundColor: '#c7e0cb' }}>
+                <Typography
+                  className="no-margin"
+                  variant="body1"
+                  sx={{ fontWeight: 700 }}
+                >
+                  Data Coordination Plan
+                </Typography>
+              </SynapsePriceTableCell>
+              <SynapsePriceTableCell sx={{ backgroundColor: '#d2e6d5' }}>
+                <Typography variant="body2" sx={{ fontWeight: 700 }}>
+                  Contact Sage**
+                </Typography>
+              </SynapsePriceTableCell>
+            </Box>
+          </Box>
+          <Accordion expanded={true}>
```

Comment:
I appreciate that the `expanded` state of these Accordion sections are not mutually exclusive...I think that's the default behavior in an old version of Bootstrap, and it drives me nuts when I can't open more than one at one time on other websites. It might just be me who is bothered by that 🙂

Reply:
I think MUI differentiates "controlled" vs "uncontrolled" components (when managed by a parent component vs local state).  https://mui.com/material-ui/react-accordion/
I used the wrong prop though, needed to use defaultExpanded!